<!-- {about} --><a name="about"></a>
  <h2 id="about" class="first">About <em>Bombyx mori</em></h2>
  <p>The silkworm, <em>Bombyx mori</em>, has been used for silk production for over 5,000 years, and continues to be an economically important species. <em>B. mori</em> is a fully domesticated insect, and due to its ease of culture, the silkworm has become a model organism in the study of lepidopteran and arthropod biology [1,2].</p>
  <p>Picture credit (Creative Commons BY-SA 3.0): <a href="http://commons.wikimedia.org/wiki/File:Silkworms3000px.jpg">Wikimedia Commons</a> 2009</p>
  <p><!--nbsp--> <!--nbsp--></p>
  <p id="about"></p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
  <p>The <em>Bombyx mori</em> genome was assembled by the International Silkworm Genome Sequencing Consortium, a joint collaboration of the National Institute of Agrobiological Sciences (NIAS) in Japan and the Southwest University in Chongqing City, China. The consortium integrated two early assemblies into a single high-quality assembly with ~10x coverage of the genome [3]. Approximately 87% of the scaffolds are assembled into 28 linkage groups; this is not represented in Ensembl Metazoa, but is available at <a href="http://sgp.dna.affrc.go.jp/KAIKObase">KAIKObase</a> [4]. In release 22 of Ensembl Metazoa (March 2014) the assembly was updated to remove contigs (primarily bacterial contaminants and duplicate regions) that were not part of the canonical INSDC record.</p>
  <p>There are two sets of scaffold names in use for <em>Bombyx mori</em>. The INSDC records have scaffold names like "<em>:_Bm_scaf586_strain:_p50T_Dazao.</em>", but these have been abbreviated in Ensembl Metazoa (from release 22 onwards), to "<em>scaf586</em>", for example. (The longer INSDC nomenclature is listed as a synonym, however, so searches and uploads using that format should still go to the right place.) These scaffold numbers are also used by KAIKObase. Earlier Ensembl Metazoa releases used the nomenclature from SilkDB, which is formatted like "<em>nscaf29</em>"; note that the numbers in these names do not correpond to those in the INSDC names. The <a href="/tools.html">assembly converter tool</a> maps between our current assembly ("GCA_000151625.1") and the earlier version ("Bmor1").</p>
  <p>In adding <em>Bombyx mori</em> to LepBase we have also included NCBI scaffold names as synonyms, to allow searches for these names as well.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p>Aggregation of <em>ab intio</em> and similarity based gene predictions was achieved using a <a href="http://glean-gene.sourceforge.net">GLEAN</a>-like algorithm resulting in over 14,000 gene models. The annotations displayed in Ensembl Metazoa are imported from <a href="http://silkworm.genomics.org.cn">SilkDB</a> [5]. Changes to the scaffolds in release 22 of Ensembl Metazoa (see Assembly section) resulted in some genes mapping to new positions; in all but two cases the DNA sequence is unchanged. The exceptions are BGIBMGA009343, which lost 8 exons that were on a duplicated region, and BGIBMGA014289 which is truncated because part of the gene was on a contig believed to be a bacterial contaminant. Non-coding RNA genes were added using the <a href="http://metazoa.ensembl.org/info/genome/genebuild/ncrna.html">Ensembl Genomes pipeline</a>, and BLAST hits and protein features have been computed.</p>
  <p>In addition to the SilkDB gene models, we have also included the NCBI RefSeq gene models and annotations in LepBase. These two sets of gene models are included as separate tracks to allow easy comparison between the two sets of predictions in any genomic region.</p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <ol>
    <li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/15355234">The genetics and genomics of the silkworm, <em>Bombyx mori</em></a>.<br>Goldsmith MR, Shimada T, Abe H. 2005. Annual Review of Entomology. 50:71-100.</li>
    <li><a id="ref-02"></a><a href="http://europepmc.org/abstract/MED/24160415">Advances in silkworm studies accelerated by the genome sequencing of <em>Bombyx mori</em></a>.<br>Xia Q, Li S, Feng Q. 2014. Annual Review of Entomology. 59:513-536.</li>
    <li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/19121390">The genome of a lepidopteran model insect, the silkworm <em>Bombyx mori</em></a>.<br>International Silkworm Genome Consortium. 2008. Insect Biochemistry and Molecular Biology. 38:1036-1045.</li>
    <li><a id="ref-04"></a><a href="http://europepmc.org/abstract/MED/19843344">KAIKObase: an integrated silkworm genome database and data mining tool</a>.<br>Shimomura M, Minami H, Suetsugu Y, Ohyanagi H, Satoh C, Antonio B, Nagamura Y, Kadono-Okuda K, Kajiwara H, Sezutsu H et al. 2009. BMC Genomics. 10:486.</li>
    <li><a id="ref-05"></a><a href="http://europepmc.org/abstract/MED/19793867">SilkDB v2.0: a platform for silkworm (<em>Bombyx mori</em>) genome biology</a>.<br>Duan J, Li R, Cheng D, Fan W, Zha X, Cheng T, Wu Y, Wang J, Mita K, Xiang Z et al. 2010. Nucleic Acids Research. 38:D453-6.</li>
  </ol>
